test/e2e/framework/.import-restrictions

```diff
@@ -1,126 +1,26 @@
 {
 	"Rules": [
-		{
-			"SelectorRegexp": "k8s[.]io/kubernetes/pkg/",

```

Comment:
You removed this rule due to "hard maintenance", but I'm not sure I understand why, can you expand on that a bit?

Reply:
@spiffxp Do you might want to keep this "k8s[.]io/kubernetes/pkg/" line?
If so, I will revert this part.

Reply:
A good reference on this may be @neolit123 's comment https://github.com/kubernetes/kubernetes/pull/80496#discussion_r306779312
long term, we would like to move the framework out of k/k so the above restriction was meant to guide us to that point.
Though right now we still have a good amount of work to do for that restriction to be valid (i think).

Reply:
I personally would prefer to keep the line, yes.  AIUI the maintenance burden serves as friction to remind us that we shouldn't be relying on pkg internal methods or types.   Instead, at least for the purposes of e2e tests, we should be using code that's meant to be exported (ie: k8s.io/utils instead of k8s.io/pkg/util)

Put another way, I think it's desirable that we document / freeze the state of e2e's importing from pkg rather than allowing more to occur.  My understanding is this rule will accomplish that for us, if I'm mistaken please let me know

Reply:
@spiffxp I see, that makes sense.
I've reverted it and added some necessary packages into the list on the latest PR.